### ISSUE TITLE
[Training problem]

### ISSUE THREAD

**Issue by BenLag2906** [zhreshold/mxnet-ssd#184]
Hi,
When I started to learn less class with SSD ( 2 classes) : It diverge quickly. A "Nan" appear on loss fonction ( L1Smooth )

Should I intiailizing my CNN differently ? because my pretrained is based on 20 classes...

Or

Is there a modification to add with my softmax function ?

Could you please give a piece of advise ?
regards
